libdnf.spec

```diff
@@ -13,7 +13,7 @@
 %global oldname libhif
 
 Name:           libdnf
-Version:        0.7.0
+Version:        0.7.1
```

Comment:
@jsilhan That's fine with me. Though, we really don't have to use crappy GNOME-style tags (`0.7.0` is fine with me).